imap_processing/lo/l1a/lo_l1a.py

```diff
@@ -126,8 +130,63 @@ def add_dataset_attrs(
         Data with attributes added.
     """
     # TODO: may want up split up these if statements into their
-    # own functions
-    if logical_source == "imap_lo_l1a_histogram":
+    #  own functions
+    # Get global attributes
+    dataset.attrs.update(attr_mgr.get_global_attributes(logical_source))
+    # Get attributes for shcoarse and epoch
+    dataset.shcoarse.attrs.update(attr_mgr.get_variable_attributes("shcoarse"))
+    dataset.epoch.attrs.update(attr_mgr.get_variable_attributes("epoch"))
+
+    if logical_source == "imap_lo_l1a_spin":
+        spin = xr.DataArray(
+            data=np.arange(0, 28, dtype=np.uint16),
```

Comment:
Is this supposed to be hard-coded? If so, you can get away with a uint8 here too.